lib/services/local/externalauditstorage.go

```diff
@@ -47,31 +47,48 @@ var (
 
 // ExternalAuditStorageService manages External Audit Storage resources in the Backend.
 type ExternalAuditStorageService struct {
-	backend backend.Backend
-	logger  *logrus.Entry
+	backend         backend.Backend
+	integrationsSvc *IntegrationsService
```

Comment:
Can't we just call `NewIntegrationsService(s.backend)` as needed?

Reply:
yeah that simplifies things thanks